Documentation/gettingstarted.rst

```diff
@@ -697,3 +698,236 @@ If instead you just want to shut down the VM but may use it later,
+`Vagrant Install Instructions <https://www.vagrantup.com/docs/installation/>`_
+or see `Download Vagrant <https://www.vagrantup.com/downloads.html>`_ for newer versions.
+
+
+Step 1: Download the Cilium Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Download the latest Cilium `source code <https://github.com/cilium/cilium/archive/master.zip>`_
+and unzip the files.
+
+Alternatively, if you are a developer, feel free to clone the repository:
+
+::
+
+    $ git clone https://github.com/cilium/cilium
+
+Step 2: Starting a VM with Cilium
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open a terminal and navigate into the top of the ``cilium`` source directory.
+
+Then navigate into ``examples/mesos`` and run ``vagrant up``:
+
+::
+
+    $ cd examples/mesos
+    $ vagrant up
+
+The script usually takes a few minutes depending on the speed of your internet
+connection. Vagrant will set up a VM, install the Docker container runtime and
+run Cilium with the help of Docker compose. When the script completes successfully,
+it will print:
+
+::
+
+    ==> default: Creating cilium-kvstore
+    Creating cilium-kvstore ... done
+    ==> default: Creating cilium ... 
+    ==> default: Creating cilium
+    Creating cilium ... done
+    ==> default: Installing loopback driver...
+    ==> default: Installing cilium-cni to /host/opt/cni/bin/ ...
+    ==> default: Installing new /host/etc/cni/net.d/10-cilium.conf ...
+  
+    $
+
+If the script exits with an error message, do not attempt to proceed with the
+tutorial, as later steps will not work properly.   Instead, contact us on the
+`Cilium Slack channel <https://cilium.herokuapp.com>`_.
+
+Step 3: Accessing the VM
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+After the script has successfully completed, you can log into the VM using
+``vagrant ssh``:
+
+::
+
+    $ vagrant ssh
+
+
+All commands for the rest of the tutorial below should be run from inside this
+Vagrant VM.  If you end up disconnecting from this VM, you can always reconnect
+in a new terminal window just by running ``vagrant ssh`` again from the Cilium
+directory.
+
+Step 4: Confirm that Cilium is Running
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Cilium agent is now running and you can interact with it using the ``cilium`` CLI client. Check the status of the agent by running ``cilium status``:
+
+::
+
+    $ cilium status
+    KVStore:            Ok
+    ContainerRuntime:   Ok
+    <...>
+    Cilium:             Ok
+
+The status indicates that all necessary components are operational.
+
+Step 5: Run Script to Start Marathon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Start Marathon inside of the vbox:
+
+::
+
+    $ ./start_marathon.sh
+
+Step 6: Start a Simple Web-Server and Client 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The web-server simply responds to requests on a particular port. Use the ``curl`` command below to launch it.
+
+::
+
+    $ curl -i -H 'Content-Type: application/json' -d @web-server.json 127.0.0.1:8080/v2/apps
+
```

Comment:
I would add a note that the output should be:
```
HTTP/1.1 201 Created
[...]
```
As the extensive response from Marathon looks a bit like something went wrong.